### ISSUE TITLE
next_set_bit implementation

### ISSUE THREAD

**Issue by michaelmior** [Ezibenroc/PyRoaringBitMap#53]
Is there an efficient way to retrieve the next bit set after a particular value similar to [`nextSetBit`](https://docs.oracle.com/javase/7/docs/api/java/util/BitSet.html#nextSetBit(int)) in Java `BitSet`s?

An example of what this might look like:

```python
>> x = BitMap([1, 2, 4])
>> x.next_set_bit(2)
2
>> x.next_set_bit(3)
4
>> x.next_set_bit(5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: No such bit set
```

**Comment by michaelmior** [comment]
On further investigation, it seems that this is implemented in CRoaring as `roaring_move_uint32_iterator_equalorlarger`, but I don't believe a direct equivalent exists in PyRoaringBitMap. Given that the CRoaring API is iterator-based, it would probably make more sense to have a method which returns a Python iterator advanced to the appropriate location. `next_bit_set` could be a simple wrapper which calls `next` on this iterator.

**Comment by lemire** [comment]
@michaelmior 


What you can do is use a roaring iterator and then if you want to skip ahead, you use the  `roaring_move_uint32_iterator_equalorlarger` function. This will work quite well.

**Comment by lemire** [comment]
> Given that the CRoaring API is iterator-based, it would probably make more sense to have a method which returns a Python iterator advanced to the appropriate location. next_bit_set could be a simple wrapper which calls next on this iterator.


Ideally, that's not how you would do it. If you care for performance, then you will expose the iterator in Python and call that instead.

**Comment by michaelmior** [comment]
@lemire That was my point. By "Python iterator" I meant a method similar to the current `__iter__`. I'll try to get together a PR if that's something you'd be willing to merge.

**Comment by lemire** [comment]
@michaelmior  Sorry that I misunderstood.

I expect that a PR will be welcomed. :-)

**Comment by Ezibenroc** [comment]
Fixed by #54.
